tests/h/services/search_index/_queue_test.py

```diff
@@ -314,13 +323,51 @@ def test_deleting_multiple_jobs_with_the_same_annotation_id(
         counts = queue.sync(len(jobs))
 
         assert counts == {
-            Queue.Result.COMPLETED_UP_TO_DATE: 2,
+            Queue.Result.COMPLETED_UP_TO_DATE.format(tag="test_tag"): 2,
+            Queue.Result.COMPLETED_TAG_TOTAL.format(tag="test_tag"): 2,
             Queue.Result.COMPLETED_TOTAL: 2,
         }
         for job in jobs:
             assert job not in db_session.query(Job)
         batch_indexer.index.assert_not_called()
 
+    def test_metrics(self, factories, index, now, queue):
+        def add_job(indexed=True, updated=False, deleted=False, **kwargs):
+            annotation = factories.Annotation()
+            factories.SyncAnnotationJob(annotation=annotation, **kwargs)
+
+            if indexed:
+                index(annotation)
+
+            if updated:
+                annotation.updated = now + ONE_WEEK
+
+            if deleted:
+                annotation.deleted = True
+
+        add_job()
+        add_job(indexed=False)
+        add_job(updated=True)
+        add_job(deleted=True)
+        add_job(tag="tag_2", force=True)
+
+        counts = queue.sync(5)
+
+        assert counts == {
+            Queue.Result.SYNCED_MISSING.format(tag="test_tag"): 1,
+            Queue.Result.SYNCED_DIFFERENT.format(tag="test_tag"): 1,
+            Queue.Result.SYNCED_TAG_TOTAL.format(tag="test_tag"): 2,
+            Queue.Result.SYNCED_FORCED.format(tag="tag_2"): 1,
+            Queue.Result.SYNCED_TAG_TOTAL.format(tag="tag_2"): 1,
+            Queue.Result.SYNCED_TOTAL: 3,
+            Queue.Result.COMPLETED_UP_TO_DATE.format(tag="test_tag"): 1,
+            Queue.Result.COMPLETED_DELETED.format(tag="test_tag"): 1,
+            Queue.Result.COMPLETED_TAG_TOTAL.format(tag="test_tag"): 2,
+            Queue.Result.COMPLETED_FORCED.format(tag="tag_2"): 1,
+            Queue.Result.COMPLETED_TAG_TOTAL.format(tag="tag_2"): 1,
+            Queue.Result.COMPLETED_TOTAL: 3,
```

Comment:
There's a bit of a toss up here between maintainability and testing. This replicates how the code does it, but this means there's no second check on the content. For example, if one of the templates gets a rogue key stroke in it as we edit, there's nothing to catch that we are doing "Compluted/Tag/Whatever".

I might err on the side of actually having the strings as they should appear. This also means the test would sort of document what we are expecting to see.